### ISSUE TITLE
parse error: Invalid numeric literal at line 1, column 5

### ISSUE THREAD

**Issue by QbaF** [thomasmauerer/hassio-addons#150]
Home Assistant 2023.5.3
Supervisor 2023.04.1
Operating System 10.1
Frontend 20230503.3 - latest

Samba Backup 5.2.0

Manual backup did well
Auto backup started at defined timw 1:30, at this same time it reported error but in category `INFO` 
`[23-05-16 01:30:13] INFO: Creating backup "Partial Backup 2023-05-16 01:30 (HA 2023.5.3)"
parse error: Invalid numeric literal at line 1, column 5`
Then he was running the backup and reported finished exactly 3 hours later 
`[23-05-16 04:30:24] INFO: Backup finished`
(should be about 10GB)

Finished but there is no backup file in shared location.

**Comment by g3r0nim00** [comment]
I have the same issue (for a while)...
Used to do Full daily backups to my QNAP via SMB share, stopped working a few weeks ago... 
`"last_backup":"2023-04-23 23:02"`

Home Assistant 2023.5.3
Supervisor 2023.04.1
Operating System 10.1
Frontend 20230503.3 - latest

Samba Backup 5.2.0

this was the logging from last nights backup run:

[23-05-15 20:00:16] INFO: Backup running ...
[23-05-15 20:00:16] DEBUG: Backups local/remote: 0/0
[23-05-15 20:00:16] DEBUG: Total backups succeeded/failed: 212/43
[23-05-15 20:00:16] DEBUG: Last backup: 2023-04-23 23:02
[23-05-15 20:00:16] DEBUG: Posting sensor data to API at /core/api/states/sensor.samba_backup
[23-05-15 20:00:16] DEBUG: API Status: 200
[23-05-15 20:00:16] DEBUG: API Response: {"entity_id":"sensor.samba_backup","state":"RUNNING","attributes":{"friendly_name":"Samba Backup","backups_local":"0","backups_remote":"0","total_backups_succeeded":"212","total_backups_failed":"43","last_backup":"2023-04-23 23:02"},"last_changed":"2023-05-15T18:00:16.969305+00:00","last_updated":"2023-05-15T18:00:16.969305+00:00","context":{"id":"01H0G93XT97WJR924RPN5HBBKX","parent_id":null,"user_id":"580292d0d60b47228d107ba3abe4bb18"}}
**[23-05-15 20:00:17] INFO: Creating backup "2023-05-15 20:00 Full 2023.5.3"
parse error: Invalid numeric literal at line 1, column 5**
[23-05-15 23:00:17] DEBUG: Posting sensor data to API at /core/api/states/sensor.samba_backup
[23-05-15 23:00:17] DEBUG: API Status: 200
[23-05-15 23:00:17] DEBUG: API Response: {"entity_id":"sensor.samba_backup","state":"FAILED","attributes":{"friendly_name":"Samba Backup","backups_local":"1","backups_remote":"0","total_backups_succeeded":"212","total_backups_failed":"44","last_backup":"2023-04-23 23:02"},"last_changed":"2023-05-15T21:00:17.487899+00:00","last_updated":"2023-05-15T21:00:17.487899+00:00","context":{"id":"01H0GKDH6FJ8HN5DVXZQ799G9K","parent_id":null,"user_id":"580292d0d60b47228d107ba3abe4bb18"}}
[23-05-15 23:00:27] DEBUG: Posting sensor data to API at /core/api/states/sensor.samba_backup
[23-05-15 23:00:27] DEBUG: API Status: 200
[23-05-15 23:00:27] DEBUG: API Response: {"entity_id":"sensor.samba_backup","state":"IDLE","attributes":{"friendly_name":"Samba Backup","backups_local":"1","backups_remote":"0","total_backups_succeeded":"212","total_backups_failed":"44","last_backup":"2023-04-23 23:02"},"last_changed":"2023-05-15T21:00:27.561185+00:00","last_updated":"2023-05-15T21:00:27.561185+00:00","context":{"id":"01H0GKDV1976S0EBMJBD4K9VTN","parent_id":null,"user_id":"580292d0d60b47228d107ba3abe4bb18"}}
[23-05-15 23:00:27] INFO: Backup finished

file was available locally but not on the SMB share... my backup is also around 10GB in size.

**Comment by g3r0nim00** [comment]
#65 
check, we both exceed the 3 hours limit. This could be the reason why it stopped working at a sudden time a while ago.
It seems there is currently no solution to overcome this 3 hours limit.

**Comment by QbaF** [comment]
Yes, it looks You are right. I found that backup in HA folder `backup`. 

> It seems there is currently no solution to overcome this 3 hours limit.

The creator of this add-on suggest to replace CLI with API calls in #79 which should definitely solve the problem. I have no idea how to bite that.

**Comment by kamcioo** [comment]
Hi @QbaF 
did You found how to replace the CLI calls with API? Also I have the problem with the 3 hours. 😤

**Comment by QbaF** [comment]
Unfortunately not. Vacation. But no help or tip from the add-on creator as well.

**Comment by realfire23** [comment]
same issue. Backup over 13 Gb, I dont understand where the 3 hours come from ?

**Comment by bighippo999** [comment]
I'm getting this error too. Only just noticed it. My backups have always taken ages but always thought it's because I set the purge to 90 days, lots of data. Doesn't really bother me as it runs during the night.

Here's the logs:
```
[23-09-24 03:05:16] INFO: Backup running ...
[23-09-24 03:05:17] INFO: Creating backup "Samba Backup 2023-09-24 03:05"
[23-09-24 06:05:05] INFO: Copying backup ed115259 (Samba_Backup_2023_09_24_03_05.tar) to share
[23-09-24 06:17:04] INFO: Deleting 7160f882 (Samba Backup 2023-09-17 03:05) local
[23-09-24 06:17:05] INFO: Deleting Samba_Backup_2023_07_26_03_05.tar on share
[23-09-24 06:17:16] INFO: Backup finished
[23-09-25 03:05:36] INFO: Backup running ...
[23-09-25 03:05:37] INFO: Creating backup "Samba Backup 2023-09-25 03:05"
parse error: Invalid numeric literal at line 1, column 5
[23-09-25 06:05:49] INFO: Backup finished
[23-09-26 03:05:09] INFO: Backup running ...
[23-09-26 03:05:11] INFO: Creating backup "Samba Backup 2023-09-26 03:05"
parse error: Invalid numeric literal at line 1, column 5
[23-09-26 06:05:40] INFO: Backup finished
[23-09-27 03:05:00] INFO: Backup running ...
[23-09-27 03:05:01] INFO: Creating backup "Samba Backup 2023-09-27 03:05"
[23-09-27 06:05:18] INFO: Copying backup d52391e4 (Samba_Backup_2023_09_27_03_05.tar) to share
[23-09-27 06:16:57] INFO: Deleting e21ea6fa (Samba Backup 2023-09-20 03:05) local
[23-09-27 06:16:57] INFO: Deleting e2c693dd (Samba Backup 2023-09-19 03:05) local
[23-09-27 06:16:57] INFO: Deleting fa768854 (Samba Backup 2023-09-18 03:05) local
[23-09-27 06:16:58] INFO: Deleting Samba_Backup_2023_07_27_03_05.tar on share
[23-09-27 06:17:09] INFO: Backup finished
[23-09-28 03:05:29] INFO: Backup running ...
[23-09-28 03:05:30] INFO: Creating backup "Samba Backup 2023-09-28 03:05"
parse error: Invalid numeric literal at line 1, column 5
[23-09-28 06:06:03] INFO: Backup finished
[23-09-29 03:05:22] INFO: Backup running ...
[23-09-29 03:05:24] INFO: Creating backup "Samba Backup 2023-09-29 03:05"
parse error: Invalid numeric literal at line 1, column 5
[23-09-29 06:05:54] INFO: Backup finished
[23-09-30 03:05:14] INFO: Backup running ...
[23-09-30 03:05:16] INFO: Creating backup "Samba Backup 2023-09-30 03:05"
parse error: Invalid numeric literal at line 1, column 5
[23-09-30 06:05:49] INFO: Backup finished
[23-10-01 03:05:09] INFO: Backup running ...
[23-10-01 03:05:11] INFO: Creating backup "Samba Backup 2023-10-01 03:05"
parse error: Invalid numeric literal at line 1, column 5
[23-10-01 06:05:40] INFO: Backup finished
[23-10-02 03:05:01] INFO: Backup running ...
[23-10-02 03:05:02] INFO: Creating backup "Samba Backup 2023-10-02 03:05"
parse error: Invalid numeric literal at line 1, column 5
[23-10-02 06:05:31] INFO: Backup finished
[23-10-03 03:05:51] INFO: Backup running ...
[23-10-03 03:05:53] INFO: Creating backup "Samba Backup 2023-10-03 03:05"
parse error: Invalid numeric literal at line 1, column 5
[23-10-03 06:06:22] INFO: Backup finished
[23-10-04 03:05:41] INFO: Backup running ...
[23-10-04 03:05:43] INFO: Creating backup "Samba Backup 2023-10-04 03:05"
parse error: Invalid numeric literal at line 1, column 5
[23-10-04 06:06:12] INFO: Backup finished
[23-10-05 03:05:32] INFO: Backup running ...
[23-10-05 03:05:33] INFO: Creating backup "Samba Backup 2023-10-05 03:05"
```

Prior to the 24th, the backups look to be running fine, though that maybe a fluke.
Looking at my backups folder, they're all around 7.5Gb. I do have some gaps during Sept, but I know I hit the network cable to the samba server and it was offline at some point.
